examples/basic/index.html

```diff
@@ -69,41 +72,55 @@
 
   // Could be defined by default by the app or the user, custom UI, external request, etc.
   var mappings = {
-    task1: {
-      common: {
-        triggerdown: {left: 'lefthand', right: 'righthand'}
-      },
-      'vive-controls': {
-        gripdown: 'changeTask',
-        trackpaddown: 'logdefault'
-      },
-      'oculus-touch-controls': {
-        abuttondown: 'changeTask'
-      },
-      'windows-motion-controls': {
-        gripdown: 'changeTask'
-      },
-      keyboard: {
-        't_up': 'logdefault',
-        'c_up': 'changeTask'
+    behaviours: {
+      default: {
+        'vive-controls': {
+          trackpad: 'dpad'
+        }
       }
     },
-    task2: {
-      'vive-controls': {
-        triggerdown: 'logtask2',
-        gripdown: 'changeTask'
-      },
-      'oculus-touch-controls': {
-        triggerdown: 'logtask2',
-        abuttondown: 'changeTask'
+    mappings: {
+      task1: {
+        common: {
+          triggerdown: {left: 'lefthand', right: 'righthand'}
+        },
+        'vive-controls': {
+          'grip.down': 'changeTask',
+          'trackpad.down': 'logdefault',
+          'trackpad.doubletouch': 'doubletouch',
+          'trackpad.doublepress': 'doublepress',
+          //'trackpad.longpress': 'longpress',
+          'trackpaddpadleftdown': 'dpadleft',
```

Comment:
could use the "down" activator here, but might be nice to have an example showing its optional. In that case though maybe add a comment explaining this.

Reply:
Yep good point, fixed